src/soap/handlers/GetSitesObject.js

```diff
@@ -48,16 +64,30 @@ export async function* getSitesObject(
         // TODO: Sort! We should default to _id. Check other services.
         $limit: 2000
       },
-      request.params && request.params.org
-        ? { organization_id: await helpers.orgId(request.params.org) }
+      organization &&
```

Comment:
Sorry for all of the crazy on this. So to clarify, the org coming in the route is the org slug in Dendra. Seems it doesn't match what Martin used. So I just need to fix the data in Dendra. Presuming the org slug should become `woftest`.

If the org is missing in the route, then stations for ALL orgs are returned. Hence the undefined in the assign. Is this a valid use of the service -- probably not. But it supports it so let's leave it for now.